### ISSUE TITLE
(WSL error at Startup) "{Application Error} ..." When I run WSL.

### ISSUE THREAD

**Issue by Msadr471** [microsoft/WSL#6344]
Hi, I get this message when I run the WSL.
![image](https://user-images.githubusercontent.com/49529241/102378466-1523fd80-3fdb-11eb-9beb-07a0b975a030.png)
I do not know exactly why, but I think the problem is from here.
![image](https://user-images.githubusercontent.com/49529241/102378596-3b499d80-3fdb-11eb-8c0a-87b6be943d0a.png)
I realized that the WSL Virtual Network Card is not here!
Do you have a solution?
I also tried this:
![image](https://user-images.githubusercontent.com/49529241/102378999-a3987f00-3fdb-11eb-822e-da086e336804.png)
I just do not want to reset the option because I installed software that can not be deleted!

Thanks.

**Comment by OneBlue** [comment]
@Msadr471: Please follow [these instructions](https://github.com/Microsoft/WSL/blob/master/CONTRIBUTING.md#reporting-issues-in-wsl) to create a feedback hub entry and submits logs.

**Comment by Msadr471** [comment]
Well, I finally made it after a thousand and one attempts!
![image](https://user-images.githubusercontent.com/49529241/102853538-c260a080-4435-11eb-93d3-5cd297e2286b.png)


See this link: https://github.com/microsoft/WSL/issues/5508#issuecomment-706052402 
I did these steps 3 times in a row until my problem was finally solved! But unfortunately, I had to use the **reset** option and delete everything! I do not know why, but maybe I could fix the problem without **resetting** it!

Thanks, @OneBlue  for your feedback but my problem was solved! Do I need to create a feed again?
